src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -0,0 +1,443 @@
+| InjectedVolumeUnit            | RECOMMENDED           | [string][]    | Unit of the injected volume of the radiotracer (for example, "mL").                                                                                                                                                                                   |
+| Purity                        | RECOMMENDED           | [number][]    | Purity of the radiolabeled compound (for example, 90).                                                                                                                                                                                                |
+
+#### Time
+
+This section is mandatory and contains timing information about the imaging experiment. We refer to the common principles for the standards for describing dates and timestamps, including possibilities for anonymization ([LINK](https://bids-specification.readthedocs.io/en/stable/02-common-principles.html)).
+
+| **Key name**    | **Requirement level** | **Data type**            | **Description**                                                                                                                                                           |
+| --------------- | --------------------- | ------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| TimeZero        | REQUIRED              | [string][]               | Time zero to which all scan and/or blood measurements have been adjusted to, in the unit “hh:mm:ss”. This should be equal to `InjectionStart` or `ScanStart`.             |
+| ScanStart       | REQUIRED              | [number][]               | Time of start of scan with respect to `TimeZero` in the default unit seconds.                                                                                             |
+| InjectionStart  | REQUIRED              | [number][]               | Time of start of injection with respect to `TimeZero` in the default unit seconds. This corresponds to DICOM Tag (0018,1042) converted to seconds relative to `timeZero`. |
+| FrameTimesStart | REQUIRED              | [array][] of [numbers][] | Start times for all frames relative to `TimeZero` in default unit seconds.                                                                                                |
+| FrameDuration   | REQUIRED              | [array][] of [numbers][] | Time duration of each frame in default unit seconds. This corresponds to DICOM Tag (0018,1242) converted to seconds.                                                      |
+| ScanDate        | RECOMMENDED           | [string][]               | Date of scan in the default unit “yyyy:mm:dd”.                                                                                                                            |
```

Comment:
for all other modalities this is to be stored in the scans.tsv file, see [here](https://bids-specification.readthedocs.io/en/bep-009/03-modality-agnostic-files.html#scans-file).

Please keep it consistent. This information is already hard enough as it is w.r.t. possible deidentification and date shifts.

Reply:
hmm, I was not totally correct here: the `acq_time` in scans.tsv is a "datetime", not only a "date". Still, I would prefer to see the date of acquisition for PET in the same place as for MRI.

Reply:
So just to be clear, there should be a new file scans.tsv and the only thing in there should be datetime since the MR spec doesn't have it?

Reply:
Well this technically is not an MR file.

But pretty much a modality agnostic file that can also be useful to know which modality was acquired when with respect to the others.

The current status of scans.tsv files is optional, but your table seems to make them recommended.

Not sure we can / want to have different levels of "requirements" for files like this depending on the modality within the dataset.

https://bids-specification.readthedocs.io/en/stable/03-modality-agnostic-files.html#scans-file

Reply:
Hmm, should we have a discussion with the maintainers regarding this? @Remi-Gau @effigies @tsalo @sappelhoff